src/styles/blocks/categories.scss

```diff
@@ -0,0 +1,71 @@
+.categories {
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+    margin-bottom: 40px;
+
+    @include onTablet {
+      row-gap: 64px;
+    }
+  }
+}
+
+.category {
+  &__photos {
+    @include pageGrid;
```

Comment:
Please make sure you have defined the `pageGrid` mixin in your SCSS, otherwise, this line will lead to a compilation error.